expectation.go

```diff
@@ -176,6 +196,31 @@ func (e *Expectation) Check(t TestingT) bool {
 	return true
 }
 
+func (e *Expectation) checkStdin(t TestingT) bool {
+	actual := string(e.readStdin)
+	switch expected := e.stdin.(type) {
+	case string:
+		if expected != actual {
+			if len(actual) <= 64 {
```

Comment:
What is the 64 about here?

Reply:
Arbitrary “shortish line length”; just avoiding megabytes of stdin being dumped into an error message.